src/layouts/RootLayout.astro

```diff
@@ -0,0 +1,42 @@
+---
+import Footer from '../components/navigation//Footer';
+
+import Header from '../components/navigation/Header';
+import '../assets/styles/styles.scss';
+
+interface Props {
+	title: string;
+}
+
+const { title } = Astro.props;
+---
+<!doctype html>
+	<html lang="en">
+		<head>
+			<meta charset="UTF-8" />
+			<meta name="description" content="Astro description" />
+			<meta name="viewport" content="width=device-width" />
+			<!-- Favicon -->
+			<link rel="icon" type="image/svg+xml" href="/assets/favicon/favicon.svg" />
+			<!-- Manifest -->
+			<link rel="manifest" href="/assets/favicon/site.webmanifest" crossorigin="use-credentials" />
+			<!-- Normalize -->
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/normalize/7.0.0/normalize.min.css">
+			<!-- Google fonts -->
+			<link rel="preconnect" href="https://fonts.googleapis.com">
+			<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+			<link href="https://fonts.googleapis.com/css2?family=Inter:wght@100..900&display=swap" rel="stylesheet">
+			<!-- Devicons -->
+			<link rel="stylesheet" type='text/css' href="https://cdn.jsdelivr.net/gh/devicons/devicon@latest/devicon.min.css" />
+			<!-- Font Awesome -->
+			<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.5.2/css/all.min.css" integrity="sha512-SnH5WK+bZxgPHs44uWIX+LLJAJ9/2PkPKZ5QiAj6Ta86w+fsb2TkcmfRyVX3pBnMFcV7oQPJkl9QevSCWr3W6A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
```

Comment:
These CDN imports should be avoided if its possible. Its always better to install npm package for this.